src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/controller/MaintainSupplierPricingController.kt

```diff
@@ -437,8 +437,5 @@ class MaintainSupplierPricingController(
     const val ADD_PRICE_EXCEPTION = "/add-price-exception"
     const val REMOVE_PRICE_EXCEPTION = "/remove-price-exception"
     const val UPDATE_PRICE = "/update-price"
-
-    fun routes(): Array<String> =
```

Comment:
All of these routes() functions are no longer needed.  There were really there for the filter which really was the wrong way round.